napari/layers/base/layer_event_handler.py

```diff
@@ -0,0 +1,26 @@
+from ...utils.event_handler import EventHandler
+from napari.utils.base_interface import BaseInterface
+
+
+class LayerEventHandler(EventHandler):
```

Comment:
Holy crap, I was going through the code above this going, "Ok, so all the complexity is going away, I bet it's just been moved to the LayerEventHandler...." But this class is so dead simple! :exploding_head: